src/clr-angular/utils/drag-and-drop/providers/drag-and-drop-event-bus.spec.ts

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright (c) 2016-2018 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+import {ClrDraggableEvent} from "../interfaces/draggable-event";
+
+import {ClrDragAndDropEventBus} from "./drag-and-drop-event-bus";
+
+export default function(): void {
+    describe("Drag And Drop Event Bus Provider", function() {
+        beforeEach(function() {
+            this.dndEventBus = new ClrDragAndDropEventBus();
+        });
+
+        it("notifyAllOnDragStart() should cast to dragStart subject", function() {
+            const testPayLoad: ClrDraggableEvent = {dragDataStore: "dragStartTest"};
+            this.dndEventBus.dragStarted.subscribe((value: any) => {
+                expect(value).toBe(testPayLoad);
+                expect(value.dragDataStore).toBe("dragStartTest");
+            });
+            this.dndEventBus.notifyAllOnDragStart(testPayLoad);
```

Comment:
This test in particular is testing the `dragStarted` Observable but I am wondering if we need a separate test which just checks if the `notify*` methods are defined.

Reply:
We basically test that by calling notify* methods :D

Reply:
Yes. I wanted to check if we need to separately test that too. Not requesting a change but just a question. Feel free to ignore this :D